dungeon/test/testingUtils/SimpleCounter.java

```diff
@@ -0,0 +1,13 @@
+package testingUtils;
+
+public final class SimpleCounter {
```

Comment:
das sieht nach einem `AtomicInteger` aus...

Reply:
@tgrothe Was genau sieht nach AtomicInteger aus? Die Klasse SimpleCounter? Die wird doch grad neu definiert?

Reply:
@cagix Ich meinte diese Klasse. Es gibt mit `AtomicInteger` ja schon eine Klasse, die hoch- und runterzählen kann. Das wäre doch dann Reinventing the wheel?

Reply:
Mag sein. Aber ist das nicht was von @malt-r?

Reply:
Ohje, ich sehe gerade, dass das eine Hilfsklasse für Tests ist... dann will ich nix gesagt haben.

Reply:
@tgrothe OK. Das ist so schräg und besch***en wie dieses gesamte Projekt: 

- Das Ding stammt von @AMatutat und @AHeinisch und war ursprünglich im jetzigen Game-Subprojekt im `test/testingUtils/`, d.h. das Ding ist seit dem 15.05.23 dabei (lt. Git). 
- @fwatermann hat das jetzt beim Splitten in Subprojekte in den entsprechenden Ordner im Dungeon-Subprojekt rüberkopiert - interessanterweise sollte das eigentlich gar nicht nötig sein, weil man ja eh die Ressourcen aus Game braucht?!

Jedenfalls ist das Ding gaaaanz tief eingegraben, sowohl in Game als auch in Dungeon. Das fummele ich hier in diesem PR aber nicht raus - das gibt ein neues Ticket.

Reply:
#1413